FirebaseGoogleAuthUI/Sources/FUIGoogleAuth.m

```diff
@@ -214,92 +234,77 @@ - (void)signInWithOAuthProvider:(FIROAuthProvider *)oauthProvider
   }];
 }
 
+- (void)requestScopesWithPresentingViewController:(UIViewController *)presentingViewController
+                                       completion:(FUIAuthProviderSignInCompletionBlock)completion {
+  GIDSignIn *signIn = [self googleSignIn];
+  [signIn addScopes:self.scopes presentingViewController:presentingViewController
+           callback:^(GIDGoogleUser *user, NSError *error) {
+    [self handleSignInWithUser:user
+                         error:error
+      presentingViewController:presentingViewController
+                      callback:^(FIRAuthCredential *credential,
+                                 NSError *error,
+                                 FIRAuthResultCallback result,
+                                 NSDictionary<NSString *,id> *userInfo) {
+      if (completion != nil) {
+        completion(credential, error, result, nil);
```

Comment:
Should this be `completion(credential, error, result, userInfo)` here?

Reply:
Done, good catch!